src/GafferScene/SceneAlgo.cpp

```diff
@@ -203,12 +201,11 @@ IECore::TransformPtr GafferScene::SceneAlgo::transform( const ScenePlug *scene,
 	}
 
 	MatrixMotionTransformPtr result = new MatrixMotionTransform();
-	ContextPtr transformContext = new Context( *Context::current(), Context::Borrowed );
-	Context::Scope scopedContext( transformContext.get() );
+	Context::EditableScope transformContext( Context::current() );
```

Comment:
Edited it to be `timeScope` in f41f7a9, since all in all other spots we're using `fooScope` rather than `fooContext`, and the types are `FooScope` rather than `FooContext`.